back/config/db.go

```diff
@@ -68,24 +68,23 @@ func InitDatabase(config Config, env string) (err error) {
 
 		database, err = gorm.Open(postgres.Open(databaseURI))
 		if err != nil {
-			return errors.Errorf("InitDatabase, unable to load database %s", err)
+			return errors.Errorf("initDatabase, unable to load database %s", err)
 		}
 
 	default:
 		database, err = InitNeonDatabase(config.Handler.VaultClient)
 		if err != nil {
-			return errors.Wrapf(err, "InitDatabase, unable to load database")
+			return errors.Wrapf(err, "initDatabase, unable to load database")
```

Comment:
Shouldn't you use one of the unable you created in the constants you created, or create a new constant